android/KMEA/app/src/main/java/com/tavultesoft/kmea/KeyboardPickerActivity.java

```diff
@@ -761,4 +761,22 @@ public void onKeyboardDownloadFinished(HashMap<String, String> keyboardInfo, int
       updateProgress.dismiss();
     }
   }
+
+  @Override
+  public void onPackageInstalled(List<Map<String, String>> keyboardsInstalled) {
+    // TODO:  Stuff for automatic updates.
+    for(Map<String, String> keyboardInfo: keyboardsInstalled) {
```

Comment:
Don't we have a similar block now in MainActivity.java?  Do we need both blocks?

Reference:  https://github.com/keymanapp/keyman/pull/520/files#diff-59bb054d4d589e6da04fc2274185a000R734